src/samples/highlights/v1/complete.json

```diff
@@ -0,0 +1,213 @@
+                            "140": "https://placehold.it/140x140"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    {
+        "title": "April/May 2016",
+        "item": {
+            "type": "podcast-episode",
+            "number": 29,
+            "title": "April/May 2016",
+            "published": "2016-05-27T13:19:42Z",
+            "image": {
+                "thumbnail": {
+                    "alt": "",
+                    "sizes": {
+                        "16:9": {
+                            "250": "https://placehold.it/250x141",
+                            "500": "https://placehold.it/500x281"
+                        },
+                        "1:1": {
+                            "70": "https://placehold.it/70x70",
+                            "140": "https://placehold.it/140x140"
+                        }
+                    }
+                }
+            },
+            "sources": [
+                {
+                    "mediaType": "audio/mpeg",
+                    "uri": "https://nakeddiscovery.com/scripts/mp3s/audio/eLife_Podcast_16.05.mp3"
+                }
+            ]
+        }
+    },
+    {
+        "title": "Lost generation",
+        "item": {
+            "type": "podcast-episode-chapter",
+            "episode": {
+                "number": 30,
+                "title": "July 2016",
+                "published": "2016-07-01T08:30:15Z",
+                "image": {
+                    "thumbnail": {
+                        "alt": "",
+                        "sizes": {
+                            "16:9": {
+                                "250": "https://placehold.it/250x141",
+                                "500": "https://placehold.it/500x281"
+                            },
+                            "1:1": {
+                                "70": "https://placehold.it/70x70",
+                                "140": "https://placehold.it/140x140"
+                            }
+                        }
+                    }
+                },
+                "sources": [
+                    {
+                        "mediaType": "audio/mpeg",
+                        "uri": "https://nakeddiscovery.com/scripts/mp3s/audio/eLife_Podcast_16.06.mp3"
+                    }
+                ],
+                "subjects": [
+                    {
+                        "id": "biochemistry",
+                        "name": "Biochemistry"
+                    },
+                    {
+                        "id": "ecology",
+                        "name": "Ecology"
+                    }
+                ]
+            },
+            "chapter": {
+                "number": 2,
+                "title": "Lost generation",
+                "time": 300,
+                "impactStatement": "Is science at risk of losing talented researchers to other professions?",
+                "content": [
```

Comment:
I agree that also for consistency with the other types we should not include `content` but rather a `podcast-episode-snippet`; this is how content types are always included when they are referred in list.

It isn't how podcast episode chapters are listed in podcast full models, but all these API definitions follow patterns of usage so if `content` and other fields are not necessary they may be dropped here.